### ISSUE TITLE
Possible accessibility improvements

### ISSUE THREAD

**Issue by sonianb** [fac25/abby-abdulla-karwan-lisa-server#41]
Here are some issues that axe DevTools flagged. 
- The error messages have insufficient colour contrast ratio. The colour contrast ratio is 2.61, whereas the expected contrast ratio is 4.5:1. 
- The document should have one `main` landmark and all page content should be contained by landmarks. [This browser extension](https://matatk.agrip.org.uk/landmarks/) might be helpful to visualise and navigate a webpage via landmarks, even if they're missing.

**Comment by sonianb** [comment]
By the way, you can use [this contrast checker](https://webaim.org/resources/contrastchecker/) to ensure that the colour contrast ratio for all the elements meets the WCAG standard minimum of 4.5:1.
